frontend/pages/deck/[id]/index.tsx

```diff
@@ -86,7 +86,7 @@ export default function Page() {
 								{{
 									cards: (
 										<CardsTab
-											deckID={deck.deckID}
+											deck={{ ...deck, deckID: deck.deckID, groupRole: group.groupRole }}
```

Comment:
```suggestion
											deck={{ ...deck, groupRole: group.groupRole }}
```
deckID is part of deck